app/experimenter/experiments/forms.py

```diff
@@ -198,6 +198,15 @@ def _get_display_name(self, field):
             return self.fields[field].label
         return field.replace("_", " ").title()
 
+    def _has_changed(self, old_status, changed_values, experiment):
+        return any(
```

Comment:
Between things evaluating to False and using `or` instead of `any([])` I think this should fit on one line?